docs/tasks/stateful-set/upgrade-from-petsets-to-stateful-sets.md

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
@kubernetes/docs should I put `assignees:` here? It's not in the template, but I assume it works for all docs?